TCMSApp/src/client/app/runs/run-execute.html

```diff
@@ -59,90 +59,212 @@
                     </div>
                 </div>
             </div>
-            <div class="text-right">
```

Comment:
What have you changed in template? can you please describe? and also why such a big commented block have appeared?